### PR TITLE
Made it so barriers appear where you are unable to slot bind to

### PR DESCRIPTION
**Pull request by NopoTheGamer**
image

### CODE REVIEW

**Review by hannibal002 — APPROVED** [review]
Does exactly what I think you want us to think this feature has to be about. I'm pleased to learn this feature exist, thanks for your PR!

**Review by romangraef — APPROVED** [review]

**Comment by jani270** [comment]
It would be cool if the slot you bind from still shows as the real item. 

image

**Review by hannibal002 — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/events/ReplaceItemEventInventory.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.moulberry.notenoughupdates.events;
+
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.item.ItemStack;
+
+public class ReplaceItemEventInventory extends NEUEvent {
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/events/ReplaceItemEventInventory.java:25]: bruh i forgot to delete that event thats useless now

**Review by Lulonaut — DISMISSED** [review]

**Comment by Lulonaut** [comment]
/home/runner/work/_temp/66d03d91-e68f-4d74-aff5-2142eb9dbe54.sh: line 3: infer: command not found

**Review by hannibal002 — APPROVED** [review]

**Review by romangraef — CHANGES_REQUESTED** [review]
image
this is still broken (binding with armor)

**Review by romangraef — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**